Applications/CLI/Tests.cmake

```diff
@@ -486,6 +486,47 @@ if(NOT OGS_USE_MPI)
         cube_1e3_expected_pcs_0_ts_101_t_1.000000.vtu cube_1e3_pcs_0_ts_101_t_1.000000.vtu displacement displacement
     )
 
+    # Mechanics; Small deformations, Ehlers (SDE)
+    AddTest(
+        NAME Mechanics_SDE_cube_1e0
```

Comment:
@wenqing good idea, @xingyuanmiao will add a DP model test in the next days. Please consider this as an additional treat, but not a reason to stall the PR. The full model is verified by the existing, more complex benchmark.